### ISSUE TITLE
Is that working?

### ISSUE THREAD

**Issue by kandebonfim** [hendricius/jekyll-quickstart#5]
Sorry, but I really need slim on gh-pages. Do you think that is possible, @hendricius? Thanks, man!

**Comment by hendricius** [comment]
Hey there,

I haven't looked at jekyll in a long time. If you like, you can open up a PR with the changes you are looking for.

**Comment by kandebonfim** [comment]
Yeah, I'll try to fix it. But I don't think I have those skills yet...
